ivc/src/ivc/columns.rs

```diff
@@ -67,7 +72,7 @@ pub type IVCPoseidonColumn = PoseidonColumn<
 ///      0       ...     34*2    76    80
 ///
 ///
-///                      Hashes
+///                      Hashes (temporarily DISABLED)
```

Comment:
It'd make more sense to also add disabling of hashes into this PR perhaps. I think cherry-picking the IVC bits is harder than folding...